x-pack/plugins/observability_ai_assistant/public/components/chat/chat_item.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import { EuiText, EuiComment } from '@elastic/eui';
+import type { AuthenticatedUser } from '@kbn/security-plugin/common';
+import { MessageRole, Message } from '../../../common/types';
+import { ChatItemAvatar } from './chat_item_avatar';
+import { ChatItemTitle } from './chat_item_title';
+import { MessagePanel } from '../message_panel/message_panel';
+import { FeedbackButtons, Feedback } from '../feedback_buttons';
+
+const roleMap = {
+  [MessageRole.User]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.userLabel',
+    { defaultMessage: 'You' }
+  ),
+  [MessageRole.System]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.systemLabel',
+    { defaultMessage: 'System' }
+  ),
+  [MessageRole.Assistant]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.assistantLabel',
+    { defaultMessage: 'Elastic Assistant' }
+  ),
+  [MessageRole.Function]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.functionLabel',
+    { defaultMessage: 'Elastic Assistant' }
+  ),
+  [MessageRole.Event]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.functionLabel',
+    { defaultMessage: 'Elastic Assistant' }
+  ),
+  [MessageRole.Elastic]: i18n.translate(
+    'xpack.observabilityAiAssistant.chatTimeline.messages.functionLabel',
+    { defaultMessage: 'Elastic Assistant' }
+  ),
```

Comment:
I don't think we need these, as it's a display issue, not a translation issue if that makes sense (in any case I think it will fail CI because you're declaring a message multiple times)

Reply:
I'll merge this in and we can fix it later.

Reply:
Oversight during copy pasting. Will fix 😃